CMakeLists.txt

```diff
@@ -71,6 +73,13 @@ elseif(VCLIBS_TARGET_ARCHITECTURE MATCHES "^arm64$")
     set(VCLIBS_I386_OR_AMD64 "arm64")
     set(VCLIBS_X86_OR_X64 "arm64")
     add_compile_definitions(_ARM64_ _VCRT_WIN32_WINNT=0x0A00 _STL_WIN32_WINNT=0x0A00)
+elseif(VCLIBS_TARGET_ARCHITECTURE MATCHES "^arm64ec$")
+    set(VCLIBS_TARGET_ARCHITECTURE "arm64ec")
+    set(VCLIBS_I386_OR_AMD64 "arm64ec")
+    set(VCLIBS_X86_OR_X64 "arm64")

```

Comment:
Double-checking, is this intentional?

Reply:
Yes it is, and it probably deserves at least a "yes, really" comment to make that clear. `VCLIBS_X86_OR_X64` is used only as the suffix of `TOOLSET_LIB`, which is roughly `%VCToolsInstallDir%lib\${VCLIBS_X86_OR_X64}`. This directory is where the linker finds the default `lib`s (libcpmt, vcruntime, etc.) and corresponding `pdb`s, and the so-called [Link Options](https://learn.microsoft.com/en-us/cpp/c-runtime-library/link-options?view=msvc-170) (`chkstk.obj`, `legacy_stdio_float_rounding.obj`). `%VCToolsInstallDir%lib\arm64ec` only contains the Link Options (which our build doesn't need). IIUC the "vanilla" ARM64 libs are used.

I'll add a comment.